cmake-tool/simulate_scripts/simulate.py

```diff
@@ -47,9 +49,7 @@ def parse_args():
 def notice(message):
     # Don't call this without initialising `progname`.
     assert(progname)
-    sys.stderr.write("{}: {}".format(progname, message))
-    sys.stderr.flush()
-
+    print("{}: {}".format(progname, message), file=sys.stderr)
```

Comment:
this change should to into the other commit, it's not related to the dry-run feature

Reply:
Maybe it's easier if you swap the commits, first fix the printing and then add the dry-run

Reply:
I don't know how to seperate the commits...

Reply:
I meant separate the change set and make two commits from this instead of one.